testdata/grepinputUN

```diff
@@ -0,0 +1,2 @@
+abcሴdef
+xyz
```

Comment:
Deliberate lack of trailing newline. I just checked in the file, as it was being created in the tests.